Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,69 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var interactionManger = new InteractionManager();
+
+        StartOver:
+            interactionManger.PerformInteractions();
+
+            Console.WriteLine("Whould you like to start again?");
+            var answer = Console.ReadLine();
+            if (answer.ToLower() == "y")
+                goto StartOver;
+        }
+    }
+
+
+
+    public class InteractionManager
+    {
+        private Person _person;
+
+        public void PerformInteractions()
+        {
+            _person = new Person
+            {
+                Name = Prompter("What is your full name?"),
+                Age = int.Parse(Prompter("What is your age?")),
+                Weight = double.Parse(Prompter("What is your weight in kg?")),
+                Height = double.Parse(Prompter("What is your height in cm?"))
+            };
+
+            Console.WriteLine($"{_person.Name} is {_person.Age} years old, his weight is {_person.Weight} kg and his height is {_person.Height} cm.");
+
+            Console.WriteLine($"{_person.Name} has a BMI of {_person.BMI}");
+        }
+
+        public string Prompter(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+    }
+
+    public class Person
+    {
+        public string Name { get; set; }
+        public int Age { get; set; }
+        public double Weight { get; set; }
+        public double Height { get; set; }
+        public double BMI { get { return CalculateBmi(); } }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename property 'BMI' to match pascal case naming rules, consider using 'Bmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)